classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @student << student
```

Comment:
- [ ] As I have asked earlier, In line 11. There should be `@students` instead of `@student`. Please fix that typo

Reply:
Thanks very much @Deepakdanger